pilot/pkg/xds/ads_test.go

```diff
@@ -80,7 +80,7 @@ func TestAgent(t *testing.T) {
 	defer tearDown()
 
 	// TODO: when authz is implemented, verify labels are checked.
-	cert, key, err := bs.CA.GenKeyCert([]string{spiffe.Identity{"cluster.local", "test", "sa"}.String()}, 1*time.Hour)
+	cert, key, err := bs.CA.GenKeyCertNoLifetimeCheck([]string{spiffe.Identity{"cluster.local", "test", "sa"}.String()}, 1*time.Hour)
```

Comment:
Looks like we are generating certs for a workload. Should be okay to not check the lifetime in this case though...

Reply:
Oh, I didn't notice this test is generating for a workload... I can write a function GenKeyCert() for this test, to make this 100% accurate.